app/models/patient.rb

```diff
@@ -246,7 +246,7 @@ def vaccinated?(programme)
     # TODO: This logic doesn't work for vaccinations that require multiple doses.
 
     vaccination_records.any? do
-      _1.administered? && _1.programme_id == programme.id
```

Comment:
I do like how straightforward this change was

Reply:
Me too!